README.md

```diff
@@ -15,15 +15,12 @@ Kubeapps is a web-based UI for deploying and managing applications in Kubernetes
 
 Use the Helm chart to install the latest version of Kubeapps: 
 
-***Note:*** Kubeapps supports both Helm 2 and Helm 3. For detailed instructions on how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).
+***Note:*** Kubeapps 2.0 and onwards supports Helm 3 only. For detailed instructions on how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).
```

Comment:
I'd be a bit more specific here? "supports the Helm 3 API only, which in most cases still allows installing charts made for Helm 2", or something similar?

Reply:
okay, TBH, most users won't know the difference between the Helm 3 API and the chart specification versioning so will try to simplify it.

Reply:
Yeah, I agree it doesn't need to mention the API, I just mean that helm 2 charts are still supported outside of corner cases. There's nothing that needs to be done to still install those charts.